sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/trace/_exporter.py

```diff
@@ -68,6 +83,15 @@ def export(self, spans: Sequence[ReadableSpan], **kwargs: Any) -> SpanExportResu
         :rtype: ~opentelemetry.sdk.trace.export.SpanExportResult
         """
         envelopes = []
+        if self._should_collect_otel_resource_metric():
```

Comment:
Osvaldo: "it only gets made when there's a breeze payload to be made"
Make sure export is not called when there are no spans. If it is called without spans, then change the code to only add the resource metric when other spans are present.